docs/Makefile

```diff
@@ -3,22 +3,34 @@
 .SILENT:
 .DEFAULT_GOAL := help
 
```

Comment:
I guess you can add a comment here saying `#If your app is written in golang, add the following lines to set GOPATH`

Reply:
Done here f258099

Reply:
Good catch, @Takehime! Thanks!